docs/strategies/circuit-breaker.md

```diff
@@ -247,6 +249,8 @@ sequenceDiagram
 
 #### Complex: unhappy path sequence diagram
 
+The circuit will break and later it will transition into `HalfOpen` state. The probe will fail so, the circuit breaker will become broken again (`Open` state).
```

Comment:
```suggestion
The circuit will break and later it will transition into the `HalfOpen` state. The probe will then fail so the circuit breaker will become broken again (the `Open` state).
```